### ISSUE TITLE
Comments on Setup section

### ISSUE THREAD

**Issue by davideklund** [ENCCS/julia-for-hpc#63]
Here are three different comments relating to the Setup section:

1. In Visual Studio press cogwheel Extension Settings to find field for Julia Executable Path.

2. To find the path to Julia on Windows one can start julia REPL, type
Sys.BINDIR

3. When installing conda, jupterlab and so on on Windows may have to extend PATH environment variable with path to conda. Can do this by:
    entering Settings 
    type "environment"
    enter "Edit the system environment variables"
    click Environment Variables
    choose Path and Edit and add your path to conda

**Comment by davideklund** [comment]
Comment pertains also to (I think):

https://enccs.github.io/julia-intro/

**Comment by wikfeldt** [comment]
why is conda needed here? Julia should be installed with [juliaup](https://github.com/JuliaLang/juliaup)

**Comment by wikfeldt** [comment]
oh right i guess this issue is referring to the usage of Jupyter. We're keeping that optional and will be using VSCode mostly

**Comment by davideklund** [comment]
Right. I think the 3rd comment is about jupyterlab installation and some issue that arose when I tried it going through the setup steps.

**Comment by davideklund** [comment]
We can close this. The first one is fixed and the other two were suggestions.
